core/dbt/tests/fixtures/project.py

```diff
@@ -310,38 +292,7 @@ def run_sql_file(self, sql_path):
 
     # run sql from a string, using adapter saved at test startup
     def run_sql(self, sql, fetch=None):
```

Comment:
Can you rename this to differentiate it from the utils `run_sql`?

Reply:
I renamed the other one, since is the one that we're actually using in the code.